CONTRIBUTING.md

```diff
@@ -51,12 +41,6 @@ Once the issue has been agreed upon and developed, you can send a pull-request.
 The pull-request should have a detailed explanation of the changes that you are doing.
 If you worked on a GitHub issue, please provide the link as part of the description.
 
-If your changes have impact on the front-end, an image showing the changed screen will be
-very helpful to understand your changes. If possible, provide a screenshot before the change
-and another one after the change. 
```

Comment:
This hasn't really been followed but I actually think it would be nice in many cases.

Reply:
I've added a comment; yes, it's implicit; we had long templates for PRs with all of this but it created more noise and we saw that people didn't follow it.